src/analyzer.rs

```diff
@@ -319,14 +337,16 @@ mod tests {
     kind: Redefinition { first_type: "alias", second_type: "alias", name: "foo", first: 0 },
   }
 
+  const UNKNOWN_ALIAS_TARGET: &str = "alias foo := bar\n";
   analysis_error! {
     name: unknown_alias_target,
-    input: "alias foo := bar\n",
+    input: UNKNOWN_ALIAS_TARGET,
     offset: 6,
     line: 0,
     column: 6,
     width: 3,
-    kind: UnknownAliasTarget {alias: "foo", target: "bar"},
+    kind: UnknownAliasTarget {alias: "foo", target: Namepath::from(Name::from_identifier(Token{
```

Comment:
This should be formatted to be on multiple lines:

```suggestion
    kind: UnknownAliasTarget {
      alias: "foo",
      target: Namepath::from(Name::from_identifier(Token {
        column: 13,
        …
```